engine/src/main/java/org/hibernate/search/engine/logging/impl/Log.java

```diff
@@ -498,4 +499,9 @@ SearchException targetFieldsSpanningMultipleNestedPaths(String fieldPath1,
 	@Message(id = ID_OFFSET + 112,
 			value = "Unable to close saved value for key %1$s: %2$s")
 	SearchException unableToCloseSavedValue(String keyName, String message, @Cause Exception e);
+
+	@LogMessage(level = Logger.Level.WARN)
+	@Message(id = ID_OFFSET + 113,
```

Comment:
I wasn't sure what's the rule on message ids with backporting... I've updated it to be the next for 6.1